src/Host/Client/Impl/Session/RSessionProvider.cs

```diff
@@ -275,25 +279,18 @@ public class RSessionProvider : IRSessionProvider {
                     await Task.WhenAll(transactions.Select(t => t.AcquireLockAsync(cancellationToken)));
                     await Task.WhenAll(transactions.Select(t => t.ReconnectAsync(cancellationToken)));
                 } catch (OperationCanceledException ex) when (!(ex is RHostDisconnectedException)) {
-                    _callback.WriteConsole(Resources.RSessionProvider_SwitchingWorkspaceCanceled.FormatInvariant(_brokerProxy.Name, GetUriString(_brokerProxy)));
```

Comment:
Ok

---
In reply to: [80278538](https://github.com/Microsoft/RTVS/pull/2393#discussion_r80278538) [](ancestors = 80278538)